content/blog/2017/12/2017-12-11-auto-convert-freestyle-jenkins-jobs-to-coded-devops-pipeline.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Auto-Convert Freestyle Jenkins Jobs to Coded DevOps Pipeline" 
+tags:
+- plugin
+- pipeline
+author: alyssat
```

Comment:
In such case @alyssat will be listed as an author in the Author box in the bottom